ext/nnstreamer/tensor_converter/tensor_converter_python3.cc

```diff
@@ -171,27 +171,32 @@ PYConverterCore::convert (GstBuffer *in_buf, GstTensorsConfig *config)
   config->rate_d = rate_d;
 
   if (output) {
+    GstTensorInfo *_info;
     unsigned int num_tensors = PyList_Size (output);
 
     out_buf = gst_buffer_new ();
     for (unsigned int i = 0; i < num_tensors; i++) {
       PyArrayObject *output_array
           = (PyArrayObject *) PyList_GetItem (output, (Py_ssize_t) i);
 
+      _info = gst_tensors_info_get_nth_info (&config->info, i);
       mem_size = PyArray_SIZE (output_array);
       mem_data = _g_memdup ((guint8 *) PyArray_DATA (output_array), mem_size);
 
       out_mem = gst_memory_new_wrapped (
           (GstMemoryFlags) 0, mem_data, mem_size, 0, mem_size, mem_data, g_free);
-      gst_buffer_append_memory (out_buf, out_mem);
+
+      gst_tensor_buffer_append_memory (out_buf, out_mem, _info);
```

Comment:
Checking if _info is invalid is done in gst_tensor_buffer_append_memory.
Is a return check required in 189 ?

Reply:
I will make new PR to validate tensors-info for python.